crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -569,8 +569,9 @@ pub fn reshape_config(node: &Node) -> Vec<i64> {
         panic!("Zero shape size is not supported");
     }
 
+    // TODO: check "shape" attribute
```

Comment:
Is this specific to an opset that's not supported yet, hence the "TODO"?

Reply:
Yes, correct. I do not have a specific use case but since we handle similar one and I noticed from the spec, we are missing it. I just added TODO as a debt, which we do not need to make a payment on.